lib/ASRouter.jsm

```diff
@@ -62,11 +62,17 @@ const TRAILHEAD_CONFIG = {
   INTERRUPTS_EXPERIMENT_PREF: "trailhead.firstrun.interruptsExperiment",
   TRIPLETS_ENROLLED_PREF: "trailhead.firstrun.tripletsEnrolled",
   BRANCHES: {
-    interrupts: [["join"], ["sync"], ["nofirstrun"], ["cards"]],
+    interrupts: [
+      ["modal_control"],
+      ["modal_variant_a"],
+      ["modal_variant_b"],
+      ["modal_variant_c"],
+      ["full_page_d"],
+      ["full_page_e"],
+    ],
     triplets: [["supercharge"], ["payoff"], ["multidevice"], ["privacy"]],
   },
-  LOCALES: ["en-US", "en-GB", "en-CA", "de", "de-DE", "fr", "fr-FR"],
-  EXPERIMENT_RATIOS: [["", 0], ["interrupts", 1], ["triplets", 3]],
+  EXPERIMENT_RATIOS: [["", 0], ["interrupts", 1], ["triplets", 0]],
```

Comment:
Do you mind confirm with Felix that we do want 100% of users in an experiment branch (including 1 branch of control)?

Reply:
NI Felix here https://bugzilla.mozilla.org/show_bug.cgi?id=1583262#c2

Reply:
Scope of this bug is specific to about:welcome interrupts experiment in 71 and doesn't touch extended triplets experiment (hold back study) in 71. 

@k88hudson should we open a new bug to decide if we want to end extended triplets experiment in 71 which means we need to remove [setupExtendedTriplets here](https://github.com/mozilla/activity-stream/pull/5330/files#diff-6201bd23e10848d056ecc572cecd3e18R1057).  This should be follow up of https://bugzilla.mozilla.org/show_bug.cgi?id=1581912

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1589146